doc-content/kogito-docs/src/main/asciidoc/creating-running/con_kogito-automation.adoc

```diff
@@ -0,0 +1,56 @@
+
+== Cloud-first priority
+
+{PRODUCT} is designed to run and scale on a cloud infrastructure. You can use {PRODUCT} with the latest cloud-based technologies, such as Quarkus, Knative, and Apache Kafka, to get fast boot times and instant scaling on container application platforms, such as {OPENSHIFT}.
+
+.Technologies used with {PRODUCT}
+image::kogito/kogito-related-technolgies.png[Image of cloud-based technologies]
+
+For example, {PRODUCT} is compatible with the following technologies:
+
+* *{OPENSHIFT}*, based on Kubernetes, is the target platform for building and managing containerized applications.
+* *Quarkus* is the new native Java stack for Kubernetes that you can use when you build applications with {PRODUCT} services.
+* *Spring Boot* is also supported with {PRODUCT} if you need to use the Spring Framework with {PRODUCT}.
+* *GraalVM* enables you to use native compilation with {PRODUCT}, resulting in fast start-up times and minimal footprint. For example, a native {PRODUCT} service starts in about 0.003ms, about 100 times faster than a non-native start-up. Fast start-up is almost a necessity in a cloud ecosystem, especially if you need small serverless applications.
+* *Knative* enables you to build serverless applications with {PRODUCT} that you can scale up or down (to zero) as needed.
+* *Prometheus* and *Grafana* are compatible with {PRODUCT} services for monitoring and analytics with optional extensions.
+* *Kafka*, *Infinispan*, and *Keycloak* are also some of the middleware technologies that {PRODUCT} uses internally.
+
+== Domain-specific flexibility
+
+{PRODUCT} adapts to your business domain instead of forcing you to modify your domain to work with {PRODUCT}. You can expose your {PRODUCT} services with domain-specific APIs, based on the processes and rules that you have defined.
+
+For example, a process for onboarding employees could generate remote REST API endpoints that you can use to onboard new employees or get information on their status, all using domain-specific JSON data.
+
+.Example custom API endpoints in Swagger
+image::kogito/kogito-domain-api-endpoints-example.png[Image of REST API endpoints in Swagger UI]
+
+You can also expose domain-specific data through events or in a data index so that the data can be consumed and queried by other services.
+
+== Developer-centered experience
+
+Another focus of {PRODUCT} is optimum developer experience. You can use much or all of your existing tooling and workflow to develop, build, and deploy {PRODUCT} services, whether locally for testing or into the cloud. Quarkus and Spring Boot offer a dev mode to help with local testing, and Quarkus also offers live reload of your processes and rules in your running applications for advanced debugging.
```

Comment:
I am not sure we should mention Spring dev mode, as we are not targeting that kind of experience specifically

Reply:
Well, as I understand it, we're really not even targeting Sprint Boot much at all, but basically tolerating it for those who already use it and need to keep it. I don't interpret this as any kind of support or target statement, as much as factual.